src/PokeList.js

```diff
@@ -4,9 +4,9 @@ import PokeItem from './PokeItem';
 class PokeList extends Component {
 
     render() {
-        const pokeData = this.props.pokemon;
-        const pokeList = pokeData.map((item , index) => {
-            return <PokeItem pokemon={item} key={index}/>
+        const { pokemon } = this.props;
+        const pokeList = pokemon.map(item => {
+            return <PokeItem pokemon={item} />
```

Comment:
just add a key in the PokeItem tag with a key of something like the pokemon id and that will get rid of your ocnsole error!